scripts/beck/st/error_injection.py

```diff
@@ -380,8 +354,13 @@ def Induce512kbError():
                     count = count + 1
     return count
 
-# This function take stob_f_container, stob_f_key and returns cob_f_cotainer, cob_f_key and device_id
+
 def ConvertAdstob2Cob(stob_f_container, stob_f_key):
+    '''
+
```

Comment:
Remove extra line

Reply:
This check was given by codacy:
1 blank line required between summary line and description (found 0) (D205)